Mon 3112 fix source install (#7160)
centreon/centreon: doc/en/installation/from_sources.rst

  $ pear upgrade-all

Debian Stretch / Ubuntu 18.04
=============================

Add the php 7.1 repository :
For Debian Stretch :

  ::

    $ apt-get install apt-transport-https lsb-release ca-certificates
    $ wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg
    $ echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" >> /etc/apt/sources.list.d/php.list
    $ apt-get update

For Ubuntu 18.04 :
.. note::

> It is necessary to add sudo in front of the following commands

Reviewer: ```suggestion
You must add sudo in front of the following commands
```

Extract the Centreon archive::

  ::

    $ tar zxf centreon-web-18.10.x.tar.gz

Change directory::

  ::
    $ cd centreon-web-18.10.x

Run the installation script::

  ::
    $ ./install.sh -i

.. note::

  The installation script allows customized configuration; this process will show you the best paths to use. Furthermore quick yes/no questions can be replied to by [y] most of the time.

.. note::

>   If centreon sources have been downloaded from github, run those commands :

Reviewer: ```suggestion
  If centreon sources have been downloaded from github, run the following commands :
```
